slime/utils/arguments.py

```diff
@@ -1639,6 +1646,16 @@ def slime_validate_args(args):
         args.prefill_num_servers is not None and args.rollout_external
     ), "prefill_num_servers cannot be set when rollout_external is set."
 
+    assert args.qkv_format in [
+        "thd",
+        "bshd",
+    ], f"qkv_format {args.qkv_format} is not supported. (only 'thd' and 'bshd' are supported)"
```

Comment:
this check doesn't seem necessary as the argparse choices should already do the check